src-symfony-compatibility/v6/Symfony/LessStrictArgvInput.php

```diff
@@ -58,15 +58,15 @@ public function getOption(string $name): mixed
         return false;
     }
 
-    protected function setTokens(array $tokens)
+    protected function setTokens(array $tokens): void
```

Comment:
The path to this file still says `v6`. Maybe this should be in a `v7` folder for clarity.